docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
ahh, I was assuming we'd run it from the project root. If we want to run from `./docker/` that works too.